engine/src/main/java/org/hibernate/validator/internal/util/privilegedactions/GetDeclaredMethodHandle.java

```diff
@@ -35,10 +35,6 @@ public static GetDeclaredMethodHandle action(Lookup lookup, Class<?> clazz, Stri
 		return new GetDeclaredMethodHandle( lookup, clazz, methodName, false, parameterTypes );
 	}
 
-	/**
-	 * Before using this method on arbitrary classes, you need to check the {@code HibernateValidatorPermission.ACCESS_PRIVATE_MEMBERS}
-	 * permission against the security manager.
-	 */
```

Comment:
Why did you remove this comment?

Reply:
Because it doesn't apply for this action. We only use it for accessing our own private field in the programmatic API (recent change). We don't need the calling code base to have a permission for that (and indeed the code location using this method isn't applying that check).

Reply:
Yeah, for now. But if we use this class for other things, we need to be careful about this. So maybe change it to a warning that you need to be aware of this issue and check the permission if need be but I wouldn't remove the comment altogether.